### ISSUE TITLE
Headings must have an appropriate relative hierarchy to other headings on the same page. 

### ISSUE THREAD

**Issue by hostle83** [DSpace/dspace-angular#3071]
**Describe the bug**
Heading levels on a page should be applied structurally and hierarchically, without skipping levels. E.g. not displaying an `<h1>` followed by an `<h3>`; `<h1>` should be followed by `<h2>` followed by `<h3>` and so forth, without skipping levels. See also: https://www.w3.org/WAI/tutorials/page-structure/headings/

**To Reproduce**
As an example, on https://sandbox.dspace.org/entities/orgunit/17f528b5-cc95-4207-81c7-b7fce3b2ccfe, heading levels are skipped

**Expected behavior**
Heading levels should not be skipped

**Related work**

**Comment by atarix83** [comment]
@tdonohue @hostle83

4science is interested in working on this task. 
We should try to fix wherever this problem is reported, not only in the mentioned item page. Estimation is about 6h
